source/Application-Starter/LeveledLogger.class.st

```diff
@@ -0,0 +1,110 @@
+Class {
+	#name : #LeveledLogger,
+	#superclass : #Object,
+	#instVars : [
+		'stdout',
+		'stderr'
+	],
+	#category : #'Application-Starter'
+}
+
+{ #category : #accessing }
+LeveledLogger class >> default [
+
+	^ self outputTo: VTermOutputDriver stdout errorsTo: VTermOutputDriver stderr
+]
+
+{ #category : #accessing }
+LeveledLogger class >> outputTo: anOutputStream errorsTo: anErrorStream [
+
+	^ self new initializeOutputTo: anOutputStream errorsTo: anErrorStream
+]
+
+{ #category : #logging }
+LeveledLogger >> error: anErrorMessage [
```

Comment:
Why not logError: ?

Reply:
Because you already are in the context of a logger, and i considered it to be redundant.
But i don't really care if we change it to log<level>